pico/services/src/main/java/io/helidon/pico/services/DefaultServices.java

```diff
@@ -375,15 +375,15 @@ void bind(
             Module module,
             boolean initializing) {
         String moduleName = module.named().orElse(module.getClass().getName());
-        boolean isLoggable = DefaultPicoServices.LOGGER.isLoggable(System.Logger.Level.INFO);
+        boolean isLoggable = DefaultPicoServices.LOGGER.isLoggable(System.Logger.Level.TRACE);
```

Comment:
This one is probably fine, but Application is more important to keep that info level.

One thought is we can key off of the global pico.debug flag to control the INFO level for this one.